gum/backend-qnx/gumprocess-qnx.c

```diff
@@ -150,6 +147,8 @@ gum_process_enumerate_threads (GumFoundThreadFunc func,
     {
       carry_on = func (&details, user_data);
     }
+
+    thread.tid += 1;
```

Comment:
We tend to use `++` elsewhere for simple increments, so might be good to do that here as well.